src/modules/pos/validator/components/CommissionHistory/CommissionHistoryRow.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+import DateTimeFromTimestamp from 'src/modules/common/components/timestamp';
+import styles from './styles.css';
+
+const CommissionHistoryRow = ({ data, index }) => {
+  const isLatest = index === 0;
+  return (
+    <div className={`${grid.row} ${styles.historyDetailsRow}`}>
+      <div className={`${grid['col-md-6']} ${styles.timestamp}`}>
```

Comment:
Borders and stuff looks strange, there is a top border but no border (when it shouldnt. bee) between the rows
<img width="868" alt="image" src="https://github.com/LiskHQ/lisk-desktop/assets/8784876/b511cede-a8b1-4789-bfa3-bcd09a7f3bd8">

Reply:
The top border was fixed in light mode but apparently not in dark mode. I'll fix that along with the border between the rows